scripts/echo_stdin-client.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# This script just aims to provide a way for ansible-vault to read standard input.
+# ansible-vault doesn't let you specify passwords on the command line -- you either
+# need to provide a password file or a password script. This functions as the latter,
+# reading passwords piped in
+read -r line
+echo $line
```

Comment:
If unquoted, `$line` will get "unpacked" into an argument vector (munging any whitespace, and possibly other special characters).
```suggestion
echo "$line"
```